package/kedro_viz/models/flowchart.py

```diff
@@ -555,13 +558,20 @@ class DataNodeMetadata(GraphNodeMetadata):
     # command to run the pipeline to this data node
     run_command: Optional[str] = field(init=False, default=None)
 
+    preview: Optional[Dict] = field(init=False, default=None)
+
     # TODO: improve this scheme.
     def __post_init__(self, data_node: DataNode):
         self.type = data_node.dataset_type
         dataset = cast(AbstractDataSet, data_node.kedro_obj)
         dataset_description = dataset._describe()
         self.filepath = _parse_filepath(dataset_description)
 
+        if self.type in PREVIEW_DATASETS:
+            # If the kedro-datasets is on the latest and does have the _preview
+            if (hasattr(dataset, '_preview')):
+                self.preview = dataset._preview(40)
```

Comment:
Why not make this configurable here?
![Uploading image.png…]()

Reply:
Based on our telemetry, hardly anyone uses this.

Reply:
That's interesting. I would argue this is a slightly more compelling thing for users to actually change, but it's also something we can wait to see if users start asking for :)

Reply:
this is now done @datajoely 😄 I just included this today

Reply:
Is it toggled or can they pass the preview number of rows?

Reply:
it is just toggled on and off at the moment, with the number of rows sets to 40